fhir-model/src/main/java/com/ibm/fhir/model/path/evaluator/FHIRPathEvaluator.java

```diff
@@ -254,20 +267,18 @@ private void reset() {
             
             Collection<FHIRPathNode> currentContext = getCurrentContext();
             if (currentContext.isEmpty()) {
-                // early exit
                 return SINGLETON_FALSE;
             } else if (currentContext.size() > 1) {
                 throw new IllegalArgumentException(String.format("Input collection has %d items, but only 1 is allowed", currentContext.size()));
             }
             
             ExpressionContext typeName = arguments.iterator().next();
-            String identifier = typeName.getText();
+            String identifier = typeName.getText().replace("`", "");
```

Comment:
double-check: are there any complicated double escapes? 
eg... java \\\ means a single instance...

Reply:
In FHIRPath the tick mark is the escape character for when special characters are used in an identifier. The visitIdentifier method takes care of this for most cases but I had to add it in here for when I get the raw text direction.